docs/getting-started/introduction.md

```diff
@@ -1,10 +1,8 @@
 ---
+title: Introduction
 slug: /
-sidebar_position: 1
 ---
```

Comment:
I've been going through other projects and looking at their docs. The Introduction or Overview is always a top level item (unless the project just doesn't have one). Why is the intro under quickstart, in our case?

Note, the documentation philosophy I based my layout suggestions on also has the intro at the top level... https://documentation.divio.com/

Reply:
We were trying to re-organize the existing content into the divio structure and I was generally looking at various CNCF project docs sites. I came across this project `https://kubevela.io/docs/` which also seems to be an adaptation of divio structure plus built on docusaurus. I found their organization of information logical..  This my chain of thinking.. While `Getting Started`, Someone reads `Introduction`, then decides to give it a try with `Installation`.. As they explore the app, they may refer to the `Features` to get additional information and then move on to other topics based on their interest/need.. To be honest, I did not think this deep while recommending this structure to Jen but the flow of information seemed smooth this way in general.